java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/Application.java

```diff
@@ -56,6 +56,20 @@
         .longOpt("server-runtimes-map")
         .desc("Mappings between supported jdk versions, server types, and docker images")
         .build());
+
+    CLI_OPTIONS.addOption(Option.builder("m")
+        .required()
```

Comment:
I suppose we can. I chose to fail early instead, since the main use case that we're supporting is building in a fixed cloudbuild pipeline where all of the arguments are supplied ahead of time anyway. Having the guarantee that these args are present ahead of time does simplify some of the logic, but not substantially. If you feel strongly that they should be optional, then I suppose we can do that.

What do you suggest instead of `EXECUTABLE_NAME`? Maybe just printing `${0}`? It's definitely less misleading, though perhaps a bit more vague.